vm_insnhelper.c

```diff
@@ -6152,6 +6152,29 @@ vm_opt_str_freeze(VALUE str, int bop, ID id)
 /* this macro is mandatory to use OPTIMIZED_CMP. What a design! */
 #define id_cmp idCmp
 
+static VALUE
+vm_opt_duparray_include_p(rb_execution_context_t *ec, const VALUE ary, VALUE target)
+{
+    if (BASIC_OP_UNREDEFINED_P(BOP_INCLUDE_P, ARRAY_REDEFINED_OP_FLAG)) {
```

Comment:
I think it's fairly unlikely this will get redefined. You should be able `UNLIKELY` here to help the compiler order code more efficiently.

Reply:
Nice catch Kevin

Reply:
Yes